fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/ContainmentIndexImpl.java

```diff
@@ -395,7 +395,7 @@ private NamedParameterJdbcTemplate getNamedParameterJdbcTemplate() {
 
     @Override
     public Stream<String> getContains(final String txId, final FedoraId fedoraId) {
-        final String resourceId = fedoraId.getResourceId();
+        final String resourceId = fedoraId.isMemento() ? fedoraId.getBaseId() : fedoraId.getFullId();
```

Comment:
I had to switch this back as `getResourceId` only deals with things that have an actual file backing them.

A get to a timemap returned both the mementos but also the containment from the Original Resource. So I switched back to this method